plugins/importtext_procedure.cc

```diff
@@ -0,0 +1,1154 @@
+/** importextprocedure.cc
```

Comment:
Also, this only works for importing tabular files, not sparse ones.  So maybe the filename should mention tabular.